pkg/agent/svid/store/service.go

```diff
@@ -105,12 +110,16 @@ func (p *store) run(ctx context.Context) error {
 
 func getPluginName(selectors []*common.Selector) (string, error) {
 	for _, selector := range selectors {
-		// selector "store:$PLUGIN_NAME" is expected
-		if selector.Type == "store" {
-			return selector.Value, nil
+		// selector "svidstore:type:$PLUGIN_NAME" is expected
+		if selector.Type == strings.ToLower(svidstore.Type) {
+			splitted := strings.SplitN(selector.Value, ":", 2)
+			if len(splitted) > 1 && splitted[0] == typeSelector {
+				return splitted[1], nil
+			}
+			return "", errors.New("store type not found in selectors")
```

Comment:
mmm, mayit not go after for? it will return false error when first one is not the type

Reply:
Good catch. I'll remove this and just keep the error that is after the loop.